charts/prometheus-blackbox-exporter/templates/_helpers.tpl

```diff
@@ -156,6 +156,11 @@ priorityClassName: "{{ . }}"
 {{- with .Values.podSecurityContext }}
 securityContext:
 {{ toYaml . | indent 2 }}
+{{- if has "NET_RAW" .Values.securityContext.capabilities.add }}
```

Comment:
I'd suggest to not make this dependent on NET_RAW, because in the end these are 2 different ways to achieve the same goal (of NET_RAW would work out of the box). 
With the ping_group_range, you don't don't need NET_RAW anymore.